### PR TITLE
gives engi borgs a holofan projector

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request

title

## Why It's Good For The Game

why shouldn't they have it

also because @Wilchenx wanted it

## Changelog

:cl:
add: Added the holofan to engi borgs
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
i did this in 5 minutes and i did not test it.

**Comment by ghost** [comment]
i have no idea why travis would fail.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -304,7 +304,8 @@
 		/obj/item/stack/sheet/rglass/cyborg,
 		/obj/item/stack/rods/cyborg,
 		/obj/item/stack/tile/plasteel/cyborg,
-		/obj/item/stack/cable_coil/cyborg)
+		/obj/item/stack/cable_coil/cyborg),
+		/obj/item/holosign_creator/atmos
```
Comment (Wilchenx) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: code/modules/mob/living/silicon/robot/robot_modules.dm:307:error: ,: expected }
code/modules/mob/living/silicon/robot/robot_modules.dm:282:error: location of top-most unmatched {

fix it by changing line 307 to
/obj/item/stack/cable_coil/cyborg,
and line 308 to 
/obj/item/holosign_creator/atmos)

**Comment by Isy232** [comment]
This is good

**Comment by PowerfulBacon** [comment]
Putting to vote

**Review by PowerfulBacon — APPROVED** [review]

**Comment by zeskorion** [comment]
Silicon powercreep. I think this pr already existed at one point...

**Comment by PowerfulBacon** [comment]
They could really use it, especially with new atmos

**Comment by thelaughingbomb** [comment]
@zeskorion yeah i PR'd the same thing a while ago, it was inactive for 24 days so i closed it and PR'd it straight to the downstream i play on

**Comment by ghost** [comment]
> Silicon powercreep. I think this pr already existed at one point...

it's a basic engineering tool what the fuck are you on about?

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**